config/initializers/services_initializer.rb

```diff
@@ -1,6 +1,7 @@
 unless Rails.env.test?
   BugzillaService.credentials = Settings.bugzilla_credentials
   GithubService.credentials   = Settings.github_credentials
+  PivotalService.credentials  = Settings.pivotal_credentials
 
-  PivotalService.credentials = Settings.pivotal_credentials
+  GithubService.logger = Sidekiq::Logging.logger
```

Comment:
The intent is that the GithubService will log into the Sidekiq logger.  The only logging we have enabled for the GithubService is the RatelimitLogger, but we could add others in the future (like request/response logging).